contributing.md

```diff
@@ -0,0 +1,30 @@
+# Contributing
+
+We love pull requests from everyone. By participating in this project, you
+agree to abide by the Wagtail [code of conduct].
+
+[code of conduct]: https://github.com/wagtail/wagtail/blob/master/CODE_OF_CONDUCT.md
+
+If you do not have write access to the repo, fork, then clone the repo:
+
+    git clone git@github.com:your-username/bakerydemo.git
+
+Set up your machine for [development]:
+
+[development]: https://github.com/wagtail/bakerydemo/blob/master/readme.md
+
+Push to your fork and [submit a pull request][pr].
+
+[pr]: https://github.com/wagtail/bakerydemo/compare/
+
+For contributors that have write access, if you create a PR that is not
+ready for review, please assign the **work in progress** label to the PR. Once ready
+for review, please remove **work in progress** label and assign the **needs review**
+label to the PR.
+
+Please use your best judgement if committing directly to master. All non-trivial development
+should be done on a branch.
```

Comment:
Would it be worth mentioned [linking commit messages and issues](https://help.github.com/articles/closing-issues-via-commit-messages/)? That way, even without a PR, we should have a sense of what's changed and why?